CMakePresets.json

```diff
@@ -21,14 +21,21 @@
       "name": "CUDA 11",
       "inherits": [ "CUDA" ],
       "cacheVariables": {
-        "CMAKE_CUDA_ARCHITECTURES": "50;52;53;60;61;62;70;72;75;80;86"
+        "CMAKE_CUDA_ARCHITECTURES": "50;52;53;60;61;62;70;72;75;80;86;87"
       }
     },
     {
       "name": "CUDA 12",
       "inherits": [ "CUDA" ],
       "cacheVariables": {
-        "CMAKE_CUDA_ARCHITECTURES": "60;61;62;70;72;75;80;86;87;89;90;90a"
+        "CMAKE_CUDA_ARCHITECTURES": "60;61;62;70;72;75;80;86;87;89;90;90a;100;100a;101;101a;120;120a"
+      }
+    },
+    {
+      "name": "CUDA 13",
```

Comment:
Latest CUDA version at time of writing is [12.8](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#cuda-12-8-release-notes)

Reply:
Minimum supported device in cuda 13 is codegen 75